lib/swarm/tracker/tracker.ex

```diff
@@ -852,16 +852,14 @@ defmodule Swarm.Tracker do
               new_meta = Map.put(old_meta, key, value)
               Registry.update(name, [meta: new_meta, clock: rclock])
               nclock = Clock.event(nclock)
+            Clock.leq(rclock, lclock) and Map.has_key?(old_meta, key) ->
```

Comment:
There is no real change here other than logging and structuring to match the remove_meta handler making it consistent and perhaps easier to follow